content/en/docs/reference/config/analysis/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Configuration Analysis Messages
+description: Documents the individual error and warning messages produced during configurarion analysis.
+weight: 40
+layout: analysis-landing
+---
+
+`istioctl` provides rich analysis of Istio configuration state in order to identity
```

Comment:
Since these are istioctl specific messages, should we:

1. Put a link to the istioctl doc?
2. Organize the material differently, so the command and it's associated messages are co-located?

Reply:
I added a link to istioctl.

I don't think we want to colocate the warning messages with istioctl, since it's likely these messages will come out in other places too, such as in CR status fields and potentially in stand-alone UIs. As we revise the docs for the istioctl command proper, we should add links back to this pages.